autoload/tsuquyomi/config.vim

```diff
@@ -105,12 +109,21 @@ function! tsuquyomi#config#tsscmd()
   return l:cmd
 endfunction
 
+function! s:system(cmd)
+  let out = ''
+  let job = job_start([&shell, &shellcmdflag, a:cmd], {'out_cb': {ch,msg->[execute("let out .= msg"), out]}, 'out_mode': 'raw'})
```

Comment:
we could do this in separate CR and wait for neovim to support lamdas.

Here is a an example https://github.com/prabirshrestha/async.vim